packages/frontend/src/components/TestSubstep/index.tsx

```diff
@@ -121,15 +121,19 @@ function TestSubstep(props: TestSubstepProps): JSX.Element {
 
   const isTestSuccessful = currentExecutionStep?.status === 'success'
 
-  const executeTestFlow = useCallback(() => {
-    executeFlow({
-      variables: {
-        input: {
-          stepId: step.id,
+  const executeTestFlow = useCallback(async () => {
+    try {
+      await executeStep({
+        variables: {
+          input: {
+            stepId: step.id,
+          },
         },
-      },
-    })
-  }, [executeFlow, step.id])
+      })
+    } catch (e) {
+      console.error(e)
+    }
```

Comment:
can i check when does this occur ah, is it more of a sanity check

Reply:
yeap